kotlin/gradle.properties

```diff
@@ -0,0 +1,7 @@
+# Gradle
+org.gradle.jvmargs=-Xmx10g -Dfile.encoding=UTF-8
```

Comment:
10g is not reasonable here, let's use `org.gradle.jvmargs=-Xmx1024m "-XX:MaxMetaspaceSize=512m"`

Reply:
Done